javascript/javascript1/week3/cactusIO.js

```diff
@@ -0,0 +1,42 @@
+console.log(`======cactusIO=====`);
+const durationLimit = 120;
+let activities = [];
+
+function addActivity(date, activity, duration) {
+    const newActivites = { date: date, activity: activity, duration: duration };
```

Comment:
There's a shorthand for this, when the property and variable name is the same:
```javascript
const newActivity = { date, activity, duration }; // this is the same as above
```


(I've changed the name of the const, because it's a single activity.